### DISCUSSION TITLE
technology/theory/README

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#36] [Announcements]
# technology/theory/README

粥里有勺糖的个人博客，记录随笔与学习笔记，大前端相关的知识，高频面试题，个人面经等

http://localhost:4000/technology/theory/README.html

**Comment by ATQQ** [comment]
测试一下

**Reply by shellRaining** [reply]
hey 大佬，最近在看你的 sugar-blog 主题源码，有一些问题不太理解，请问可以开设一个 QA  discussion 吗，或者是微信联系您呢

**Reply by ATQQ** [reply]
@shellRaining  see 👁️  https://github.com/ATQQ/sugar-blog/discussions/78
